source/Core/Drivers/OLED.hpp

```diff
@@ -105,7 +105,7 @@ class OLED {
   static void    setBrightness(uint8_t contrast);
   static void    setInverseDisplay(bool inverted);
   static int16_t getCursorX() { return cursor_x; }
-  static void    print(const char *string, FontStyle fontStyle); // Draw a string to the current location, with selected font
+  static void    print(const char *string, FontStyle fontStyle, uint8_t num = 0); // Draw a string (or only a number of chars from string if specified) to the current location, with selected font

```

Comment:
`num` (or `n` in `.cpp`) is a fairly awkward parameter name and isnt self-documenting. What about `maxLength` as the argument?

Reply:
Ok, how about just `length` not to overstretch the declaration? Will it be fine?

Reply:
Well its more of a max limiter than an actual length; but if you keep the comment noting its a max, that should be fine.~